pipelines/pipeline_katello_36_to_37_upgrade.yml

```diff
@@ -103,44 +105,44 @@
   roles:
     - bats
 
-# Upgrade to Puppet 5
+# Upgrade server+proxy to Puppet 5
 # http://projects.theforeman.org/projects/foreman/wiki/Upgrading_from_Puppet_4_to_5#Prepare-And-Run-The-Installer
 - hosts:
   - pipeline-katello-3.6-3.7-centos7
   - pipeline-proxy-3.6-3.7-centos7
   become: true
   vars:
     puppet_repositories_version: 5
+    katello_repositories_version: 3.7
```

Comment:
This shouldn't be needed because it was previously set up.

Reply:
Shouldn't we do the Puppet 5 upgrade as part of the 3.6 -> 3.7 katello server as well then? We can do the same below with the proxy upgrade. This whole step was a workaround for an issue there was in the 3.5 migration. This should be fixed now.